.github/workflows/publish_stable.yml

```
@@ -0,0 +1,58 @@
+        with:
+          ref: master
+          fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
+      - name: Setup Python
+        uses: actions/setup-python@v6
+        with:
+          python-version: "3.11"
+      - name: Install Build Tools
+        run: |
+          python -m pip install build wheel
+      - name: version
+        run: echo "::set-output name=version::$(python setup.py --version)"
```

actionlint (1.7.10)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)